TM1py/Services/ProcessService.py

```diff
@@ -82,6 +82,48 @@ def get_all_names(self, **kwargs) -> List[str]:
         processes = list(process['Name'] for process in response.json()['value'])
         return processes
 
+    def search_string_in_body(self, search_string: str, **kwargs) -> List[str]:
+        """ Ask TM1 Server for list of process names that contain string anywhere in body
+
+        :param search_string: case insensitive string to search for
+        """
+        url = format_url("/api/v1/Processes?$select=Name&$filter=" \
+                         "contains(toupper(PrologProcedure),toupper('{}')) " \
+                         "or contains(toupper(MetadataProcedure),toupper('{}')) " \
+                         "or contains(toupper(DataProcedure),toupper('{}')) " \
+                         "or contains(toupper(EpilogProcedure),toupper('{}'))",
+                         search_string, search_string, search_string, search_string
+            )
+        response = self._rest.GET(url, **kwargs)
+        processes = list(process['Name'] for process in response.json()['value'])
+        return processes
+
+    def search_string_in_name(self, name_startswith: str = None, name_contains: Iterable = None,
```

Comment:
I think it would be a nice feature if it was configurable whether to search with a logical `and` or a logical `or` like what you did in https://github.com/cubewise-code/tm1py/commit/5b3386e14a381f277ee0f2859cf093a476f88edd

Reply:
agreed, i can make that update

Reply:
updated: https://github.com/cubewise-code/tm1py/pull/658/commits/07ace1f769e3e412b0e343e5faa065c39df61921